src/Configuration/Model/FixerConfig.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GrumPHP\Configuration\Model;
+
+/**
+ * @psalm-immutable
+ */
+class FixerConfig
+{
+    /**
+     * @var bool
+     */
+    private $enabled;
+
+    /**
+     * @var bool
+     */
+    private $fixByDefault;
+
+    public function __construct(
+        bool $enabled,
+        bool $fixByDefault
+    ) {
+        $this->enabled = $enabled;
+        $this->fixByDefault = $fixByDefault;
+    }
+
+    /**
+     * @param array{fix_by_default: bool, enabled: bool} $config
+     */
+    public static function fromArray(array $config): self
+    {
+        return new self(
+            (bool) ($config['enabled'] ?? false),
```

Comment:
Not sure if casting is needed here since static analyser will prevent that. Can stay for extra level of security though

Reply:
Yeah, the configuration component also makes sure it has the correct type.
However, I am gonna keep it as-is since I like to be sure.